packages/env/README.md

```diff
@@ -8,25 +8,52 @@ Ensure that Docker is running, then:
 
 ```sh
 $ cd /path/to/a/wordpress/plugin
-$ npx wp-env start
+$ npm -g i @wordpress/env
+$ wp-env start
 ```
 
 The local environment will be available at http://localhost:8888.
 
-## Instructions
-
-### Installation
+## Prerequisites
 
 `wp-env` requires Docker to be installed. There are instructions available for installing Docker on [Windows 10 Pro](https://docs.docker.com/docker-for-windows/install/), [all other versions of Windows](https://docs.docker.com/toolbox/toolbox_install_windows/), [macOS](https://docs.docker.com/docker-for-mac/install/), and [Linux](https://docs.docker.com/v17.12/install/linux/docker-ce/ubuntu/#install-using-the-convenience-script).
 
-After confirming that Docker is installed, you can install `wp-env` globally like so:
+`wp-env` is also distributed through NPM (Node Package Manager), so Node.js and NPM are required. The latest LTS version of Node.js is used to develop `wp-env` and is recommended.
```

Comment:
We require Node because `wp-env` is written in Node. Saying its because its distributed through NPM is kind of ambivalent.

Reply:
There was an issue a couple weeks ago where a user ran wp-env but their node version was much older than what we support. I think this change is trying to clarify our supported node version for people consuming wp-env from outside the Gutenberg repo?

Reply:
Yeah, but the wording feels odd.

>`wp-env` is also distributed through NPM (Node Package Manager), so

That can be omitted.

Reply:
I can simplify, though for someone reading this on Github, it feels unusual to progress to the first commands that use NPM, without mentioning it as a prerequisite.

Reply:
We are still mentioning it right?